src/Paket/Program.fs

```diff
@@ -93,13 +93,16 @@ try
             trace <| results.Usage("paket config")
         else
             let args = results.GetResults <@ ConfigArgs.AddCredentials @> 
-            let source = args.Item 0
-            let username = 
-                if(args.Length > 1) then
-                    args.Item 1
-                else
-                    ""
-            Paket.ConfigFile.askAndAddAuth(source)(username)
+            if args.Length = 0 then
+                trace <| results.Usage("paket config")
+            else
+                let source = args.Item 0
+                let username = 
+                    if(args.Length > 1) then
+                        args.Item 1
+                    else
+                        ""
+                Dependencies.Locate().AddCredentials(source, username)
```

Comment:
This changed the behavior, former to this change you could run `paket config` without having a `paket.dependencies` file, now `Dependencies.Locate()` will complain about not having it.